examples/Overlay.js

```diff
@@ -113,7 +113,7 @@ const OverlayExample = React.createClass({
           target={ props => findDOMNode(this.refs.target)}
         >
           <ToolTip>
-            I'm placed to the: <strong>{this.state.placement}</strong>
```

Comment:
This isn't actually the JSX mode per se – there is a GH JSX mode but it's a bit wonky.